Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -48,6 +49,43 @@
 #if PLATFORM(IOS_FAMILY)
 #import <UIKit/UIColor.h>
 #endif
+@interface WKPDFContextMenuHandler: NSObject {
+    WebKit::UnifiedPDFPlugin* _unifiedPdfPlugin;
+}
+- (id)initWithUnifiedPDFPlugin:(WebKit::UnifiedPDFPlugin *)plugin;
+@end
+
+
+@implementation WKPDFContextMenuHandler
+
+- (id)initWithUnifiedPDFPlugin:(WebKit::UnifiedPDFPlugin *)plugin
+{
+    if (!(self = [super init]))
+        return nil;
+
+    _unifiedPdfPlugin = plugin;
+    return self;
+}
+
+- (void) openWithPreview
+{
+}
+
+- (NSMenu *) createContextMenu: (NSUserInterfaceLayoutDirection) direction
```

Comment:
Much of the style here is wrong, please consult the WebKit style guide. Mostly you've added a lot of spaces in places.